### DISCUSSION TITLE
why my github code block background color is blue permanently.

### DISCUSSION THREAD

**Discussion by mtayyab03** [community/community#53049] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

why my github code block background color is blue permanently. an issue occur on my all github pages ,i think its a bug arise , please github team solve it as soon as possible

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by StefanShivarov** [comment]
Try playing around with the Theme Preferences settings of your Github profile.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mtayyab03 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by mbechev** [reply]
Hi @queenofcorgis,

There is a similar discussion here:
https://github.com/orgs/community/discussions/52885

The issue seems to be hit by more and more people. Is there any resolution or information that the team is aware of?
